codespell_lib/data/dictionary.txt

```diff
@@ -1815,6 +1817,9 @@ analyticaly->analytically
 analyticly->analytically
 analyzator->analyzer
 analyzies->analysis, analyses, analyzes,
+analzye->analyze
+analzyed->analyzed
+analzying->analyzing
```

Comment:
Can you add the GB (s) variants of these too, as separate typo-correction pairs.

Reply:
69e4305